job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/step/MultiJobStepContextSpec.groovy

```diff
@@ -238,16 +259,40 @@ class MultiJobStepContextSpec extends Specification {
         then:
         with(context.stepNodes[0]) {
             name() == 'com.tikal.jenkins.plugins.multijob.MultiJobBuilder'
-            children().size() == 3
+            children().size() == 4
             phaseName[0].value() == 'test'
             continuationCondition[0].value() == condition
+            executionType[0].value() == 'PARALLEL'
             phaseJobs[0].value().empty
         }
 
         where:
         condition << ['FAILURE', 'ALWAYS']
     }
 
+    def 'call phase with supported execution type'(String execution) {
+        setup:
+        jobManagement.isMinimumPluginVersionInstalled('jenkins-multijob-plugin', '1.22') >> true
+
+        when:
+        context.phase('test') {
+            executionType execution
+        }
+
+        then:
+        with(context.stepNodes[0]) {
+            name() == 'com.tikal.jenkins.plugins.multijob.MultiJobBuilder'
+            children().size() == 4
+            phaseName[0].value() == 'test'
+            continuationCondition[0].value() == 'SUCCESSFUL'
+            executionType[0].value() == execution
+            phaseJobs[0].value().empty
+        }
+
+        where:
+        execution << ['SEQUENTIALLY']
```

Comment:
do not use a `where` section if there is only one option, but you could also test both supported options here